src/View/mainpage/MenuBar.py

```diff
@@ -1,451 +1,527 @@
 		:param main_window:
 		 the window of the main page
 		"""
-		self.main_window = main_window
+        self.main_window = main_window
 
-	def windowing_handler(self, state, text):
-		"""
+    def windowing_handler(self, state, text):
+        """
 		Function triggered when a window is selected from the menu.
 		:param state: Variable not used. Present to be able to use a lambda function.
 		:param text: The name of the window selected.
 		"""
-		# Get the values for window and level from the dict
-		windowing_limits = self.main_window.dict_windowing[text]
-
-		# Set window and level to the new values
-		self.main_window.window = windowing_limits[0]
-		self.main_window.level = windowing_limits[1]
-
-		# Update the dictionary of pixmaps with the updated window and level values
-		self.main_window.pixmaps = get_pixmaps(self.main_window.pixel_values,
-											   self.main_window.window, self.main_window.level)
-		self.main_window.dicom_view.update_view()
-
-	def anonymization_handler(self):
-		"""
+        # Get the values for window and level from the dict
+        windowing_limits = self.main_window.dict_windowing[text]
+
+        # Set window and level to the new values
+        self.main_window.window = windowing_limits[0]
+        self.main_window.level = windowing_limits[1]
+
+        # Update the dictionary of pixmaps with the updated window and level values
+        self.main_window.pixmaps = get_pixmaps(
+            self.main_window.pixel_values,
+            self.main_window.window,
+            self.main_window.level,
+        )
+        self.main_window.dicom_view.update_view()
+
+    def anonymization_handler(self):
+        """
 		Function triggered when the Anonymization button is pressed from the menu.
 		"""
-		SaveReply = QtWidgets.QMessageBox.information(self.main_window, "Confirmation",
-													  "Are you sure you want to perform anonymization?",
-													  QtWidgets.QMessageBox.Yes, QtWidgets.QMessageBox.No)
-		if SaveReply == QtWidgets.QMessageBox.Yes:
-			self.main_window.hashed_path = self.main_window.callClass.runAnonymization(self.main_window)
-			self.main_window.pyradi_trigger.emit(self.main_window.path, self.main_window.filepaths,
-												 self.main_window.hashed_path)
-		if SaveReply == QtWidgets.QMessageBox.No:
-			pass
-
-	def transect_handler(self):
-		"""
+        SaveReply = QtWidgets.QMessageBox.information(
+            self.main_window,
+            "Confirmation",
+            "Are you sure you want to perform anonymization?",
+            QtWidgets.QMessageBox.Yes,
+            QtWidgets.QMessageBox.No,
+        )
+        if SaveReply == QtWidgets.QMessageBox.Yes:
+            self.main_window.hashed_path = self.main_window.callClass.runAnonymization(
+                self.main_window
+            )
+            # now that the radiomics data can just get copied across... maybe skip this?
+            RadiomicsReply = QtWidgets.QMessageBox.information(
```

Comment:
this is the one code block in MenuBar.py that was actually changed.
the rest was automated reformatting by black